LCD Interfacing/LCD.c

```diff
@@ -0,0 +1,95 @@
+#include "LCD.h"

```

Comment:
At the start of each file, let's mention what the file does and the author and description of it. This can be done before every function definition as well.

Reply:
We could. But I have left it out.